com.vivaldi.Vivaldi.yaml

```diff
@@ -0,0 +1,136 @@
+app-id: com.vivaldi.Vivaldi
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+base: org.chromium.Chromium.BaseApp
+base-version: '22.08'
+command: vivaldi
+separate-locales: false
+finish-args:
+  - --device=all # Webcams and others
+  - --env=DCONF_USER_CONFIG_DIR=.config/dconf # For GNOME proxy resolution
+  - --env=GIO_EXTRA_MODULES=/app/lib/gio/modules # For GNOME proxy resolution
+  - --env=GSETTINGS_BACKEND=dconf # For GNOME proxy resolution
+  - --env=GTK_PATH=/app/lib/gtkmodules
+  - --filesystem=/run/.heim_org.h5l.kcm-socket
+  - --filesystem=home/.local/share/applications:create # Allow for PWAs
+  - --filesystem=home/.local/share/icons:create # Allow for PWAs
+  - --filesystem=xdg-desktop
+  - --filesystem=xdg-download # Default download directory
+  - --filesystem=xdg-run/dconf # For GNOME proxy resolution
+  - --filesystem=xdg-run/pipewire-0
+  - --filesystem=~/.config/dconf:ro # For GNOME proxy resolution
+  - --filesystem=~/.config/google-chrome:ro # Import bookmarks and other data from a classic Chrome install
+  - --filesystem=~/.config/kioslaverc # For KDE proxy resolution (KDE5 only)
+  - --filesystem=~/.local/lib:create # Extra FFMpeg saved here
+  - --filesystem=~/.mozilla/firefox:ro # Import bookmarks and other data from a classic FF install
+  - --own-name=org.mpris.MediaPlayer2.chromium.*
```

Comment:
is this really what they register?

Reply:
By they do you mean Vivaldi (I should add I work for Vivaldi) and which line are you reffering to exactly. I presume the ~/.local/lib if so yes.

Reply:
> Not sure how ambiguous this looks ![image](https://user-images.githubusercontent.com/114441/272939885-c3686383-3c7d-4156-af6a-56b9f0c1dcae.png)
> 
> But yeah is this the mpris name Vivaldi register?

On my end it shows four lines in your review, I assume that's why.

Reply:
Sorry for the late reply. I had other things I needed to do. Coming back to this now. Yes like for Opera (on flathub) ours appears to be hardcoded. Perhaps I can get that fixed. In the mean time, yeah we need it like this I guess.